godel_process_execution/include/godel_process_execution/abb_blend_process_service.h

```diff
@@ -0,0 +1,38 @@
+#ifndef ABB_BLEND_PROCESS_SERVICE_H
+#define ABB_BLEND_PROCESS_SERVICE_H
+
+#include <ros/ros.h>
+#include <godel_msgs/BlendProcessExecution.h>
+
+namespace godel_process_execution
+{
+
+class AbbBlendProcessExecutionService
```

Comment:
While using sufficiently descriptive names is helpful I think this class name is longer that it needs to be.